chain/epoch-manager/src/proposals.rs

```diff
@@ -95,6 +78,7 @@ mod old_validator_selection {
 
     use crate::proposals::find_threshold;
 
+    #[allow(dead_code)]
```

Comment:
Delete all code in `mod old_validator_selection`. Delete `proposals_to_epoch_info` function and change call sites to directly call `proposals_to_epoch_info` from `crate::validator_selection`.